client/landing/stepper/declarative-flow/internals/steps-repository/domain-transfer-intro/styles.scss

```diff
@@ -4,6 +4,12 @@
 $font-family: "SF Pro Text", $sans;
 $heading-font-family: "SF Pro Display", $sans;
 
+:root {
+	--wp-components-color-accent: var(--studio-blue-50);
+	--wp-components-color-accent-darker-20: var(--studio-blue-60);
+	--wp-components-color-accent-darker-10: var(--studio-blue-60);
+}
```

Comment:
Can this be done globally?

Reply:
Made this fix on Stepper level. It seems a couple of flows were affected by [updating `@wordpress/components`](https://github.com/Automattic/wp-calypso/pull/78711).

p1689919222911329-slack-C05CT832K2T